### DISCUSSION TITLE
Do sunrise times work reliably?

### DISCUSSION THREAD

**Discussion by krampster** [gopro/labs#204] [Q&A]
I'm using the QRControl app on the phone to set a sunrise. But when I scan it, it tells me that it's going to start recording at 21:38 hours. Sunset is also wrong. I've scanned it while outside for awhile to get GPS lock, and set the current clock accurately. Is there anything I can do to debug?
!r-1800NmTr4p.10fW!S
I'm in the Seattle area.

So happy I found this though. I did a manual time start for the first test this morning. So cool. Love this feature.

**Comment by dnewman-gpsw** [comment]
Set the time manually via https://gopro.github.io/labs/control/precisiontime/ to make sure it is set correctly.

The !sAction can be wrong by a few minutes or so, and it is a little more wrong the further from the equator, although Seattle is not North enough to cause an issue. Sometimes the location is set wrong, so it easiest to just set the location manually.  This is for Seattle https://gopro.github.io/labs/control/set/?cmd=oMLLTZ=47.6,-122.3,-8 (using LLTZ, for Latt, Long, and Time Zone.)  Now !sN reports tomorrow's sunrise is at 7:51am, when google has 7:47am (this is the typical error.)  The same error is pretty constant throughout the year, so it makes daily timelapses over a year work well.   This is a year long timelapse (or mine) using this technique https://www.instagram.com/p/B9fuVXWnV_c/
